Tiltfile

```diff
@@ -515,8 +495,6 @@ include_user_tilt_files()
 
 load_provider_tiltfiles()
 
-prepare_all()
```

Comment:
We still need this. This should not be dropped. If we drop the `tilt-prepare` command will never run. 

I tried this locally and tilt fails with the following error:
> Error: open /Users/kakaraparthy/workspace/go/src/sigs.k8s.io/cluster-api/.tiltbuild/yaml/clusterctl.crd.yaml: no such file or directory

This is becasue `tilt-prepare` never ran and the artifacts needed were never created.

Note: To replicate this you will need to delete your local `.tiltbuild` folder.

Reply:
added back after checking